runelite-client/src/main/java/net/runelite/client/plugins/regenmeter/RegenMeterPlugin.java

```diff
@@ -149,5 +155,13 @@ else if (currentHP > maxHP)
 			// Show it going down
 			hitpointsPercentage = 1 - hitpointsPercentage;
 		}
+		else if (!messageSend && maxHP > currentHP && config.regenWarning() > 0 && config.regenWarning() < 100 && ((Math.round(hitpointsPercentage * 100)) == config.regenWarning() || (Math.round(hitpointsPercentage * 100)) == config.regenWarning() - 1))
+		{
+			Player local = client.getLocalPlayer();
+			notifier.notify("[" + local.getName() + "] has reached the regeneration threshold!");
```

Comment:
regeneration threshold sounds too debug-like, maybe [ username ] HP is about to regen

or something like that?

Reply:
Yeah good idea, pushed now.